app/pulp/app/settings.py

```diff
@@ -165,18 +165,22 @@
             'simple': {'format': 'pulp: %(name)s:%(levelname)s: %(message)s'},
         },
         'handlers': {
-            'stream': {
+            'pulp': {
```

Comment:
Well the idea behind renaming it was that it uses `pulp: %(name)s:%(levelname)s: %(message)s` formatter and anybody can from the config just change the logging class and use whatever they want without need to create one on their own. But it was just an Idea.